SSH tunnel connection: PostgreSQL and Redshift (#1163)
rudderlabs/rudderstack-docs: docs/destinations/warehouse-destinations/postgresql.mdx

- **Password**: Enter the password you set for the above user.
- **Port**: Enter the port number associated with your PostgreSQL instance.
- **Namespace**: Enter the schema name where RudderStack will create all the tables. If you don't specify any namespace, RudderStack will set this to the source name, by default.

### SSH connection

<div class="infoBlock">
This feature is available only for the <a href="https://www.rudderstack.com/pricing/">Growth and Enterprise plan</a> users.
</div>

> SSH tunneling is a method of transporting data over an encrypted SSH connection. You can use it to add encryption to your legacy applications as well as achieve compliance such as HIPAA, PCI-DSS, and other standards without having the need to modify the existing applications.

Reviewer: Can we change it to `transferring` ?

Reviewer: transporting -> transferring

> SSH tunneling is a method of transporting data over an encrypted SSH connection. You can use it to add encryption to your legacy applications as well as achieve compliance such as HIPAA, PCI-DSS, and other standards without having the need to modify the existing applications.

Reviewer 2: ```suggestion
SSH tunneling is a method of transporting data over an encrypted SSH connection. You can use it to add encryption to your legacy applications and achieve compliance like HIPAA, PCI-DSS, and other standards, without having to modify the existing applications.
```

> SSH tunneling is a method of transferring data over an encrypted SSH connection. You can use it to add encryption to your legacy applications as well as achieve compliance such as HIPAA, PCI-DSS, and other standards without having the need to modify the existing applications.

Reviewer 2: ```suggestion
SSH tunneling is a method of transferring data over an encrypted SSH connection. You can use it to add encryption to your legacy applications and achieve compliance with the regulations like HIPAA, PCI-DSS, etc., without having to modify the existing applications.
```

> RudderStack lets you connect to your PostgreSQL database securely over an SSH connection. Enter the following details to do the same:

Reviewer 2: ```suggestion
RudderStack lets you connect to your PostgreSQL database securely over an SSH connection by configuring these settings:
```

<img src="../../assets/warehouse-destinations/ssh-connection.png" alt="Redshift edit configuration" />

> - **SSH Connection**: Toggle on this boolean setting to use the SSH connection while connecting to your PostgreSQL database.

Reviewer 2: ```suggestion
- **SSH Connection**: Enable this setting to use the SSH connection while connecting to your PostgreSQL database.
```

- **SSH Host**: Enter the IP address of your bastion host.
- **SSH Port**: Enter the port for the above host.

> - **SSH User**: Enter the user name of your PostgreSQL database.

Reviewer: `Enter the username used to access the bastion host`

> - **SSH Public Key**: Copy the public key provided in this field and add it to the `authorizated_keys` file on your bastion host. Rudderstack will use the private key corresponding to this public key to establish the connection successully.

Reviewer 2: ```suggestion
- **SSH Public Key**: Copy the public key provided in this field and add it to the `authorizated_keys` file on your bastion host. RudderStack uses the private key corresponding to this public key to establish the connection successfully.
```

Reviewer 2: Should it be `authorized_keys`?

> 

Reviewer 2: ```suggestion

#### Enabling SSH connection for existing destinations

```

Reviewer 2: IMO, making this a subsection will be helpful. WDYT?

Author: This is common to all the source and destination settings, like not an important piece of info even, and just a one-liner. I think its fine if we dont include a header.

> You can also enable the SSH connection for an existing PostgreSQL destination by navigating to the destination's **Configuration** tab, selecting **Edit configuration** and enabling the **SSH connection** setting.

Reviewer 2: ```suggestion

#### Enabling SSH connection for existing destinations

To enable the SSH connection for an existing PostgreSQL destination, navigate to the destination's **Configuration** tab, select **Edit configuration** and enable the **SSH connection** setting.
```

> You can also enable the SSH connection for an existing PostgreSQL destination by navigating to the destination's **Configuration** tab, selecting **Edit configuration** and enabling the **SSH connection** setting.

Reviewer 2: ```suggestion
You can also enable the SSH connection for an existing PostgreSQL destination by navigating to the destination's **Configuration** tab, selecting **Edit configuration** and enabling the **SSH connection** setting.

#### Enabling SSH connection for existing destinations

```
